src/tools/wasmdump.cc

```diff
@@ -177,22 +183,25 @@ int main(int argc, char** argv) {
     result = read_binary_objdump(data, size, &s_objdump_options);
     if (WABT_FAILED(result))
       goto done;
-    s_objdump_options.print_header = 0;
+    s_objdump_options.print_header = false;
+    s_objdump_options.log_stream = nullptr;
```

Comment:
I don't understand why you set `.log_stream = nullptr` only after the second pass through `read_binary_objdump`; that way, some output is duplicated when calling `wabtdump -h --debug hw.wasm`